project/Build.scala

```diff
@@ -205,8 +238,9 @@ object Scrooge extends Build {
     name := "scrooge-generator",
     libraryDependencies ++= Seq(
       "org.apache.thrift" % "libthrift" % libthriftVersion,
-      "com.github.scopt" %% "scopt" % "3.3.0",
-      "com.github.spullara.mustache.java" % "compiler" % "0.8.18",
```

Comment:
ah, sorry, to be clear I meant move internally.  we need to do it ourselves because we have to maintain two build systems, the internal monorepo, and the public sbt.  sorry!